test/controllers/checkYourAnswers.controller.test.ts

```diff
@@ -45,7 +45,7 @@ describe('CheckYourAnswersController', () => {
   })
 
   describe('GET - ensure that page loads correctly', () => {
-    it('render single director on page', async () => {
+    it('render correct rows for single signing director', async () => {
```

Comment:
could you add an assert here to confirm that the on behalf row is not displayed for a signing director?